lang/src/main/java/org/apache/shiro/lang/util/ClassUtils.java

```diff
@@ -252,7 +274,7 @@ public Class loadClass(String fqcn) {
             ClassLoader cl = getClassLoader();
             if (cl != null) {
                 try {
-                    clazz = cl.loadClass(fqcn);
+                    clazz = Class.forName(fqcn,false,cl);
```

Comment:
I'd like to see spaces here and a comment why we changed this, e.g.
// SHIRO-767: Use Class.forName instead of cl.loadClass(), as byte arrays would fail otherwise.

Reply:
Would you please add the whitespace? Then the code would be consistent again (whitespace after parameter comma). Thanks!

Reply:
> Would you please add the whitespace? Then the code would be consistent again (whitespace after parameter comma). Thanks!

OK.I have added whitespace in the latest commit.